contrib/mod_tls.c

```diff
@@ -758,76 +758,77 @@ static X509_STORE *tls_crl_store = NULL;
 static array_header *tls_tmp_dhs = NULL;
 static RSA *tls_tmp_rsa = NULL;
 
-static void tls_exit_ev(const void *, void *);
-static int tls_sess_init(void);
```

Comment:
The declaration for `tls_sess_init` also exists already on line 797; we don't need it here as well.